### PR TITLE
2023-03-24 Tom Miles

### PR DESCRIPTION
**Pull request by allenwyma**

### CODE REVIEW

File: _episodes/interviews/2023-03-24-tom-miles.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: "CAI-NC with Tom Miles"
```
Comment (alphastrata) [inline _episodes/interviews/2023-03-24-tom-miles.md]: ai-nc

Comment (alphastrata) [inline _episodes/interviews/2023-03-24-tom-miles.md]: AI-NC

Comment (alphastrata) [reply _episodes/interviews/2023-03-24-tom-miles.md]: seems to of already been fixed? (maybe GH on mobile sucks...)

File: _episodes/interviews/2023-03-24-tom-miles.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: "AI-NC with Tom Miles"
+date: 2023-03-24T15:37:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-03-24-tom-miles.mp3
+duration: "55:31"
+length: "53292722"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Tom Miles](https://www.linkedin.com/in/thomasgregorymiles/), CTO of [AI-NC](https://www.ai-nc.com/), a platform written in Rust to get instant manufactorability feedback for your hardware designs.
```
Comment (jonhoo) [inline _episodes/interviews/2023-03-24-tom-miles.md:9]:
```suggestion
Allen Wyma talks with [Tom Miles](https://www.linkedin.com/in/thomasgregorymiles/), CTO of [AI-NC](https://www.ai-nc.com/), a platform written in Rust to get manufactorability feedback for hardware designs.
```

File: _episodes/interviews/2023-03-24-tom-miles.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- [@1:23] - AI-NC's company history
+- [@2:46] - What made Tom start a machine shop company
+- [@8:17] - AI-NC's goal to provide automation, design, and advanced manufacturing
+- [@17:04] - Communication between designers and assembly people
+- [@19:22] - The margin for error in manufacturing
+- [@30:40] - What made the team switch to Rust vs other languages
+- [@35:06] - Libraries that AI-NC uses in their services
+- [@42:35] - Integrations with third-party chat applications
+- [@46:03] - Hiring Rust talent
+- [@50:23] - Upskilling people who don’t have Rust experience
```
Comment (jonhoo) [inline _episodes/interviews/2023-03-24-tom-miles.md:32]:
```suggestion
- [@50:23] - Teaching people Rust
```

File: _episodes/interviews/2023-03-24-tom-miles.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+- [@53:27] - Advice for companies that want to use Rust in their services and platform
```
Comment (jonhoo) [inline _episodes/interviews/2023-03-24-tom-miles.md:33]:
```suggestion
- [@53:27] - Advice for companies that want to use Rust
```

**Review by jonhoo — APPROVED** [review]

File: _episodes/interviews/2023-03-24-tom-miles.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+title: "AI-NC with Tom Miles"
+date: 2023-03-24T15:37:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2023-03-24-tom-miles.md:3]:
```suggestion
date: 2023-03-24T15:23:00Z
```

File: _episodes/interviews/2023-03-24-tom-miles.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+title: "AI-NC with Tom Miles"
+date: 2023-03-24T15:23:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2023-03-24-tom-miles.md:3]:
```suggestion
date: 2023-03-24T15:31:00Z
```
had to go bump the CloudFlare cache

**Pull request merged**